content/en/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -39,6 +39,10 @@ request a particular class. Administrators set the name and other parameters
 of a class when first creating VolumeSnapshotClass objects, and the objects cannot
 be updated once they are created.
 
+{{< note >}}
+The CRDs  installations are the responsibility of the Kubernetes distribution.Respective CRD's must be apply for  API object. As the API objects `VolumeSnapshot`,`VolumeSnapshotClass` and `VolumeSnapshotContent` are not the part of Core API.
```

Comment:
Taking inspiration from the similar note [here](https://kubernetes.io/docs/concepts/storage/volume-snapshots/#introduction), do you think we can simplify this to something like:

```suggestion
Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` below will not get created.  
```

WDYT?

Reply:
@RinkiyaKeDad yes we could wait from some one from sig-storage to approve, and i have added a commit as you suggested.
Thanks

Reply:
@RinkiyaKeDad whom could we ping regarding review of this PR from sig/storage